pkg/action/hooks.go

```diff
@@ -135,6 +136,13 @@ func (cfg *Configuration) deleteHookByPolicy(h *release.Hook, policy release.Hoo
 		if len(errs) > 0 {
 			return errors.New(joinErrors(errs))
 		}
+
+		//wait for resources until they are deleted to avoid conflicts
+		if kubeClient, ok := cfg.KubeClient.(kube.InterfaceExt); ok {
+			if err := kubeClient.WaitForDelete(resources, timeout); err != nil {
+				return err
+			}
+		}
```

Comment:
On line 135, `cfg.KubeClient.Delete(resources)` is executed. It looks like this is trying to delete the resources twice. Shouldn't the code only do this once?

Reply:
Hi @mattfarina, `WaitForDelete` does not do any deletion, it waits for resources to be deleted (in case the deletion takes time, like the case of volumes...etc).